src/main/java/org/sufficientlysecure/htmltextview/LocalLinkMovementMethod.java

```diff
@@ -0,0 +1,68 @@
+package org.sufficientlysecure.htmltextview;
+
+import android.text.Layout;
+import android.text.Selection;
+import android.text.Spannable;
+import android.text.method.LinkMovementMethod;
+import android.text.method.Touch;
+import android.text.style.ClickableSpan;
+import android.view.MotionEvent;
+import android.widget.TextView;
+
+/**
```

Comment:
Remove this comment and add a proper Apache license comment above the whole file like in the other files of this library